geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java

```diff
@@ -87,12 +88,12 @@ public void initialize() {
     graphicsOverlay.getGraphics().add(sectorGraphic);
 
     // create green symbols for each sector output geometry type
-    sectorFillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, 0x8800FF00, null);
-    sectorLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0x8800FF00, 3);
-    sectorMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0x8800FF00, 3);
+    sectorFillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, Color.web("lime", 0.8), null);
```

Comment:
for readability and reduced repetition, let's use a local variable to store the lime color

```java
    Color transparentLime = Color.web("lime", 0.8);
    sectorFillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, transparentLime, null);
    sectorLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, transparentLime, 3);
    sectorMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, transparentLime, 3);

```